turingschool-examples/webpack-starter-kit
# Update webpack config and dependencies to latest versions so that app…

…lication can start correctly.

## Changes Made
- Updated `webpack`, `webpack-cli`, and `webpack-dev-server` to latest dependencies.
- A few updates had to be made to the config.js including updating the properties of the `devServer` to match the documentation.  I also removed the `sourceMapFilename` property as this seemed to be causing errors anytime a change was made to the code while the server was running.

## Why the change?
- When installing the dependencies before and trying to start the application, it would break due to some webpack errors. (See attached screenshot below)
![Screenshot 2024-01-09 at 8 58 59 AM](https://github.com/turingschool-examples/webpack-starter-kit/assets/25714149/9068d5c5-0eb9-442d-8af2-fb6854c94f29)

## How to test?
- Make sure that you can install the dependencies and start the application with no errors. 
- Run the test suite to make sure all tests still pass.
- Experiment with making a few changes to the application such as writing additional code, adding some new files, etc. to make sure students can work and build on the existing application with no problems.
